### DISCUSSION TITLE
Can't create users in web dashboard

### DISCUSSION THREAD

**Discussion by JoshMacleod** [supabase/supabase#29115] [Questions]
I've just created a new project and have been trying to add a user in the web dashboard but I'm getting an error saying: "Failed to create user: An error has occurred: Failed to fetch"

Looking in the console, the error is further explained as:
"Access to fetch at 'https://[xxx].supabase.co/auth/v1/admin/users' from origin 'https://supabase.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled."

Not entirely sure what I can do to fix this given that I've only been interacting with the web interface, would be very keen to hear any suggestions!

**Comment by GaryAustin1** [comment]
Try restarting your browser and another browser.  
Any errors in the Auth or Postgres logs?

**Reply by JoshMacleod** [reply]
Tried a different browser and still getting the same result. Nothing notable in any of the logs.

**Comment by GaryAustin1** [comment]
You might check here:
![image](https://github.com/user-attachments/assets/12c4db67-b219-4f40-b119-17163c8ea664)

I'm assuming if your logs are not getting errors then you are not getting to auth, so an auth.users trigger function you may have added as part of user management is not the issue.

You can try restarting the instance.

Otherwise contacting support would be my suggestion without anymore info.
